Dockerfile

```diff
@@ -1,13 +1,19 @@
 # Build on top of a Node.js + npm image.
-FROM node:latest
+FROM node:lts
 
 # Make a codebase directory and work in it
 RUN mkdir codebase
 WORKDIR /codebase
 
 # Load the package.json and package-lock.json files
-COPY package.json .
-COPY package-lock.json .
+COPY package.json ./
+COPY package-lock.json ./
```

Comment:
What effect does adding the / have on the image?

Reply:
On some Linux distros, using just the dot throws an error. I remember this from when I was trying out [Manjaro](https://manjaro.org/) a few months back. Hence, I added the slashes as a contingency here

Reply:
Oh, okay. That's nice.